sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -20,12 +20,26 @@ namespace Azure { namespace Storage { namespace Blobs {
   {
   };
 
+  /**
+   * @brief Specifies HTTP options for conditioanal requests based on tags.
+   */
+  struct TagAccessConditions
+  {
+    /**
+     * @brief Optional SQL statement to apply to the tags of the Blob. Refer to
+     * https://docs.microsoft.com/en-us/rest/api/storageservices/specifying-conditional-headers-for-blob-service-operations#tags-predicate-syntax
+     * for the format of SQL statements.
+     */
+    std::string TagConditions;
```

Comment:
Please create a work item to do some convenience enhancement to provide functionality for customers to construct the tag condition query string.

Reply:
language alignment is one of our most sought in Track2. Is there any source that suggests they're going to do so in any other language?